### PR TITLE
Wendy Chen's HW01

### PR DESCRIPTION
**Pull request by Wendychenn**
I added my introduction in the README.md firstly, and described what I found hard then. Actually, the syllabus is so perfect that it's quite hard to miss the key point. What I find useful is learning how to write with Markdown language, because I'm using a paper management softeware named Zotero where I can use Markdown to take notes. Though the cheatsheets is helpful, I still need time to get familar with the hot keys!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,24 @@
...
+  
+  I don't install my Git at the default location at first so I fail to connect a local RStudio project to a GitHub repo. I uninstall the Git after that, and install it correctly. But it shows *fatal: remote origin already exists*, when I mistype the url command at the beginning.
+  
+  What I find is that "origin" is the default name Git gives to the server I'm using. Because I'm cloning my repository, I need to link *origin* to my url correctly.
+  
+  [The website to know git remote](https://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes)
+  
+  
+   I open a new project, submit the right url and get the cloning done successfully.
+  ![image of success](https://fiverr-res.cloudinary.com/images/q_auto,f_auto/gigs/128537975/original/5958e62b86a1661db95e8769b149b71161e4a7b2/send-you-encouraging-message-everyday.jpg)
```
Comment (wanitchayap) [inline README.md:15]: Awwww ❤️ 💟  I understand your feeling. The first time I learn about git/github, I was very confused, and working with them was very challenging at first 😭

**Review by wanitchayap — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |    X       |              |            |
| **Coding strategy**         |       X    |              |            |
| **Presentation**            |           |      X        |            |
| **Achievement, creativity** |     X      |              |            |
| **Reproducibility**         |     X      |              |            |

Remarks:
* Great job!
* For future reference, no need to commit `.gitignore` and `.Rproj`
* Frequent commit + descriptive messages 👍

**Review by noahttesfaye — COMMENTED** [review]
I really liked your bio Wendy! I thought your graphic was really cool, and I really liked how you were able to both explain your challenges in learning about GitHub, as well as how you're solving them. I thought that it could have been a bit better to maybe share some more about your interests that led you to UC and a PhD, as well as maybe sharing a bit more about why you wanted to take the class. But aside from the content, I really thought your code was simple and hit all of the points for the assignment.

**Review by AdrianDP1 — COMMENTED** [review]
Well done:
1) Spacing: the spacing between your text looks aesthetic and clean, and it doesn't have a confusing bunch of sentences in a large paragraph
2) Good use of differently-sized headers using ##
3) Those lines before and after your title look really fancy and official-looking - a creative use of Markdown

Improvements:
1) Perhaps do something to shorten URLs? This is super minuscule and doesn't even appear in the final product, but working with a super long image URL could potentially cause problems later if edits are necessary.
2) Github Desktop (https://desktop.github.com/) might help with pulling, committing, and pushing on your computer, but no pressure to use it if you're comfortable your way now
3) Already mentioned elsewhere, but more info on your work/research?

Good luck with the class and lab!

**Pull request closed without merging**